lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
@bmeurer here is just a few examples of apply plugins being used in two separate classes that extend Tapable.

https://github.com/webpack/webpack/blob/990563f9a2b3fb3e817b7e1b9d6841bcd1eaa51e/lib/Compilation.js#L48-L676

https://github.com/webpack/webpack/blob/990563f9a2b3fb3e817b7e1b9d6841bcd1eaa51e/lib/Parser.js#L1045-L1141